spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/JvmMetricsAutoConfigurationTests.java

```diff
@@ -43,41 +49,53 @@
 
 	@Test
 	void autoConfiguresJvmMetrics() {
-		this.contextRunner.run(
-				(context) -> assertThat(context).hasSingleBean(JvmGcMetrics.class).hasSingleBean(JvmMemoryMetrics.class)
-						.hasSingleBean(JvmThreadMetrics.class).hasSingleBean(ClassLoaderMetrics.class));
+		this.contextRunner.run((context) -> validateBaseJvmMetricsBeansArePresent());
```

Comment:
Shouldn't the code be called like below for it to actually run?
```
this.contextRunner.run(validateBaseJvmMetricsBeansArePresent());
```

Reply:
Ha! Yeh that would help if we actually ran the validation. Good catch. I will update.